### PR TITLE
Updates to SA model: parameter registration, infinite wall distances, and sa_airfoil example

### PR DESCRIPTION
**Pull request by vikramvgarg**

### CODE REVIEW

**Comment by roystgnr** [comment]
I swear I've pointed @vikramvgarg to http://chris.beams.io/posts/git-commit/ twice...

At least the granularity is (modulo whitespace fixes) good now.

:+1:

**Comment by roystgnr** [comment]
We're holding off on considering merging this until all the commits are in.

File: src/physics/src/inc_navier_stokes_base.C [hunk 1]
```diff
@@ -57,21 +57,27 @@ namespace GRINS
...
-    this->_mu.init(system); 
-   
+
+    this->_mu.init(system);
+
     return;
   }
 
   template<class Mu>
+  libMesh::Real IncompressibleNavierStokesBase<Mu>::get_viscosity_value(AssemblyContext& context, unsigned int qp) const
```
Comment (roystgnr) [inline src/physics/src/inc_navier_stokes_base.C]: Did you miss committing the declaration for this?

**Comment by roystgnr** [comment]
This compiles fine for me now, but I'm seeing O(1e-5) errors from the regression tests.  Solver seemed to converge just fine, and running in serial instead of parallel didn't seem to change anything, so I'm not sure what's up.

**Comment by roystgnr** [comment]
Sorry, errors from sa_2d_turbulent_channel_regression.sh, not tests plural.  Was ff6e4d4e9 a bugfix?  If we're expecting the old "gold" solutions to be invalid then we also need to update them.

**Comment by vikramvgarg** [comment]
Yes, that was a bugfix. What output are you getting from the
sa_2d_turbulent_channel_regression test ?

On Mon, Aug 31, 2015 at 11:42 PM, roystgnr <notifications@github.com> wrote:

> Sorry, errors from sa_2d_turbulent_channel_regression.sh, not tests
> plural. Was ff6e4d4
> <https://github.com/grinsfem/grins/commit/ff6e4d4e990a71a69368f441fb5c939bb7061b5c>
> a bugfix? If we're expecting the old "gold" solutions to be invalid then we
> also need to update them.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grinsfem/grins/pull/299#issuecomment-136584753>.
>



-- 
Vikram Garg
Postdoctoral Associate
Predictive Engineering and Computational Science (PECOS)
The University of Texas at Austin
http://web.mit.edu/vikramvg/www/

http://vikramvgarg.wordpress.com/
http://www.runforindia.org/runners/vikramg

**Comment by roystgnr** [comment]
I'll send you the files in the morning.  Does "make check" still work for you?

**Comment by roystgnr** [comment]
The bugfix was that we're supposed to compute the turbulent nu from the underlying laminar nu, but you were inadvertently computing the turbulent nu from a nested turbulent nu computation?

(this is the sort of thing that needs to go in commit messages, by the way!!)

If I'm understanding that right, then the right thing to do is upload a new "gold standard" with the corrected solution.

If you can do that I'd appreciate it; I've got a kid's checkup this morning.  If you don't get around to it (but you do confirm my interpretation of that commit) by noonish I'll take care of the fix then.

**Comment by roystgnr** [comment]
Okay, this works now.  I want to get rid of the intermediate garbage though, and "rebase -i" is another step up from "add --patch", so I'll make a copy and resubmit a new PR with that.

**Comment by roystgnr** [comment]
Replaced by #300

**Pull request closed without merging**